src/serializer/modules.js

```diff
@@ -240,9 +240,9 @@ class ModuleTracer extends Tracer {
               this.nestedRequiredModulesAndValues.push({ id: moduleIdValue, value: result });
             }
             if (isTopLevelRequire) {
-              for (let { id, value } of this.nestedRequiredModulesAndValues) {
+              /*for (let { id, value } of this.nestedRequiredModulesAndValues) {
                 this.modules.initializedModules.set(id, value);
-              }
+              }*/
```

Comment:
Its the new code that figures out when modules are initialized or not, but it doesn't quite work, I'll uncomment it in the next PR.